src/GitHub.UI.Reactive/Controls/Validation/UserErrorMessages.cs

```diff
@@ -46,7 +46,7 @@ public UserErrorMessages()
             };
         }
 
-        public static readonly DependencyProperty IconMarginProperty = DependencyProperty.Register("IconMargin", typeof(Thickness), typeof(UserErrorMessages), new PropertyMetadata(new Thickness(0,10,7,0)));
+        public static readonly DependencyProperty IconMarginProperty = DependencyProperty.Register("IconMargin", typeof(Thickness), typeof(UserErrorMessages), new PropertyMetadata(new Thickness(0,0,8,0)));
```

Comment:
Just tried `RepositoryCreationControl` on master and the error doesn't even seem to appear, which is probably why nobody's noticed it's off-center :)